### DISCUSSION TITLE
Changed file content

### DISCUSSION THREAD

**Discussion by bharathalekhya** [spaceuptech/space-cloud#1657] [General]
In repository if any user changed the file content in particular file how to avoid the user to unchange the content in that file.
I

**Comment by YourTechBud** [comment]
I'm sorry which repository are we talking about?

**Reply by bharathalekhya** [reply]
Github Repository

**Reply by YourTechBud** [reply]
I'm not sure if i follow but i see two scenarios which are possible:

### 1. Restricting who can make changes on GitHub

This can be controlled through GitHub directly. SC can't really do anything here.

### 2. Use GitHub as the single source of truth for SC

You may want to make sure that config coming from GitHub cannot be changed from SC ui.

This is currently not possible in SC. However in the next release, we will support GitOps which will kinda make this possible as well. You'll simply have to wait for the next release for this

**Reply by bharathalekhya** [reply]
If we clone a git hub code locally using hooks we can able to achieve or not & we change the file content locally also it needs do work.
